app/Http/Controllers/API/ChannelsController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Http\Controllers\API;
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use App\Channels;
+use Illuminate\Broadcasting\Channel;
+use App\Http\Resources\ServiceResource;
+use SebastianBergmann\Environment\Console;
+
+class ChannelsController extends Controller
+{
+    public function store(Request $request, $id){
+        $request['user_id'] = $id;
```

Comment:
```suggestion
        $request['user_id'] = auth()->id();
```

Sempre que precisar acessar o usuário "dono" da rquisição, usar `auth()->user()->id` (ou o atalho: `auth()->id()`).